src/Microsoft.ML.Data/Transforms/TermEstimator.cs

```diff
@@ -3,15 +3,17 @@
 // See the LICENSE file in the project root for more information.
 
 using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
 using Microsoft.ML.StaticPipe;
 using Microsoft.ML.StaticPipe.Runtime;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 
-namespace Microsoft.ML.Runtime.Data
+namespace Microsoft.ML.Transforms.CategoricalTransforms
 {
-    public sealed class TermEstimator : IEstimator<TermTransform>
+    public sealed class ValueToKeyMappingEstimator: IEstimator<TermTransform>
```

Comment:
Can we add an xml summary for this class?

Reply:
we haven't even started fixing our xml docs.. I added the reference where they match.. 
Added for this one. 

---
In reply to: [228952793](https://github.com/dotnet/machinelearning/pull/1328#discussion_r228952793) [](ancestors = 228952793)